<launch>

    <!-- MoveBaseSeq node launch and Points Sequence ROS Parameters Loading -->
    
        <!--Run move_base_seqs node-->
       
            
        <node pkg="simple_navigation_goals" type="move_base_seq.py" name="move_base_seq" output="screen" respawn ="true" respawn_delay="15">
            
            <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="p_seq"> ["2","1.11","0","57.5","1.11","0","58.0","1.265","0","58.5","1.73","0","59","2.5","0","59","10","0"]
                    </rosparam>
            <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="yea_seq">["0","0","30","60","90","90"]</rosparam>
                
        </node>
        
            <!--<node pkg="simple_navigation_goals" type="turtlebot_stop.py" name="turtlebot_obstacle" output="screen"></node>-->
    
    </launch>
